resource/resource.go

```diff
@@ -342,6 +342,25 @@ func (r *Resource) find(ctx context.Context, q *query.Query, forceTotal bool) (l
 	return
 }
 
+// Count implements Storer interface.
```

Comment:
Count is not part of the Storer interface.

Reply:
My bad, will fix.